### PR TITLE
Project UI: Modifying Admin screen

### PR DESCRIPTION
**Pull request by b2simms**
Testing to see if this is the correct way to modify the screens to use assignment content.

Commit: Modifying Admin screen to use assignment_content generic layout/view.

To test: http://0.0.0.0:3000/en/admins

Output:

Before screenshot:
before

After screenshot:
after

### CODE REVIEW

**Comment by b2simms** [comment]
To test the assignment_content layout standardization:
1. admins/index.html - http://127.0.0.1:3000/en/admins
2. automated_tests/student_interface - [student] http://127.0.0.1:3000/automated_tests/7/student_interface?locale=en
3. course_summaries - http://127.0.0.1:3000/en/course_summaries
4. grade_entry_forms - http://127.0.0.1:3000/en/grade_entry_forms/3/grades
5. graders - (upload and download modal) - http://127.0.0.1:3000/en/assignments/7/graders
6. groups - http://127.0.0.1:3000/en/assignments/2/groups
7. marks_graders - http://127.0.0.1:3000/en/grade_entry_forms/3/marks_graders
8. peer_reviews - http://127.0.0.1:3000/en/assignments/11/peer_reviews
9. section - http://127.0.0.1:3000/en/sections
10. tags - http://127.0.0.1:3000/en/assignments/2/tags
11. tas - http://127.0.0.1:3000/en/tas

**Comment by b2simms** [comment]
I also tried to standardize the download modal. It is not as clean as I would like it to be; there is room for improvement. Three are three general parts to each modal: the title, the main content, and the action buttons such as download and upload.

File: app/views/assignments/index.html.erb [hunk 1]
```diff
@@ -45,6 +43,7 @@
 <% end %>
 
 
+
```
Comment (david-yz-liu) [inline app/views/assignments/index.html.erb]: Undo this change

File: app/views/grade_entry_forms/student_interface.html.erb [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-  <h1><%= t('grade_entry_forms.grades.student_title') + ' ' + @grade_entry_form.short_identifier %></h1>
-</div>
+<% content_for :title do %>
+  <%= t('grade_entry_forms.grades.student_title') + ' ' + @grade_entry_form.short_identifier %>
+<% end %>
 
 <div class='wrapper'>
   <% if @grade_entry_form.grade_entry_items.empty? %>
-    <p><%= t('grade_entry_forms.students.no_results') %></p>
+      <p><%= t('grade_entry_forms.students.no_results') %></p>
```
Comment (david-yz-liu) [inline app/views/grade_entry_forms/student_interface.html.erb]: Undo these indentation changes.

**Comment by david-yz-liu** [comment]
@b2simms good work. A few general comments:

1. For the `:title` content, make sure there's no wrapping `h1` tag around the contents.
2. Always use 2 spaces for indentation.
3. For the `@heading_buttons`, change the style to the following:

```
<% @heading_buttons = [
  { link_text: t(:add_new_admin),
    link_path: new_admin_path }
] %>

4. Make sure every file ends with a newline
5. You can inline the one-line `title`s.

Also, your `modal` layout is the right idea, but should probably not contain references to "download". Feel free to add styles. But more importantly, please revert the modal changes and do work on this in a separate branch.
